server/src/index.ts

```diff
@@ -1,7 +1,11 @@
 import './pre-start'; // Must be the first import
 import app from '@server';
 import logger from '@shared/Logger';
+import { schedule } from './utils/retry';
 
+// start retry cron job
+logger.info('Schedule retry cron job...');
+schedule();
```

Comment:
> @JFU-GIT I don't think this cron/queue approach is really ideal for exponential backoff... I think we should do something more like in this article: https://advancedweb.hu/how-to-implement-an-exponential-backoff-retry-strategy-in-javascript/
> 
> Specifically the solution listed under: Rejection-based retrying for async/await syntax

@JFU-GIT @nbragdon I agree that the use of node-cron scheduler and queuing seems complicated for the purpose.

I just read through the article and agree that the "rejection-based retrying" method seems like it is simpler and a more typical/standard way to implement this.  @nbragdon Have you used this type of method successfully in the past?

Reply:
a in place retry added.